napari/plugins/_builtins.py

```diff
@@ -69,6 +69,30 @@ def napari_write_image(path: str, data: Any, meta: dict) -> Optional[str]:
         return path
 
 
+@napari_hook_implementation(trylast=True)
+def napari_write_labels(path: str, data: Any, meta: dict) -> Optional[str]:
+    """Our internal fallback labels writer at the end of the plugin chain.
+
+    Parameters
+    ----------
+    path : str
+        Path to file, directory, or resource (like a URL).
+    data : array or list of array
+        Image data. Can be N dimensional. If meta['rgb'] is ``True`` then the
+        data should be interpreted as RGB or RGBA. If ``meta['multiscale']`` is
+        ``True``, then the data should be interpreted as a multiscale image.
+    meta : dict
+        Image metadata.
+
+    Returns
+    -------
+    path : str or None
+        If data is successfully written, return the ``path`` that was written.
+        Otherwise, if nothing was done, return ``None``.
+    """
+    return napari_write_image(path, np.asarray(data, dtype='uint32'), meta)
```

Comment:
Hmm, dunno about hardcoding the dtype, vs specifying a minimum of int32. e.g. if labels are uint64, I don't think we want to downcast!

Reply:
I suggest `dtype = data.dtype if data.dtype.itemsize >= 4 else np.uint32`. (You could be even fancier with testing for negative values, but I'm not as concerned about that.)

Reply:
I did that after some trial and error with Fiji ... it seemed that even with bioformats, the uint64 wasn’t working (image was entirely 0s).  But I agree it’s not nice to downcast... but slightly worried about people trying to open in Fiji

Reply:
implemented your suggestion

Reply:
Ah, interesting. Well, if you want, I'd be ok with checking that the max value fits in uint32. But that will be hard with dask-y things.

Reply:
anyway, happy with a merge as its stands

Reply:
Yeah let’s worry about that once someone gets confused with cross compatibility